File: wear/src/main/java/com/pdyjak/powerampwear/music_browser/artists/ArtistViewHolder.java
package com.pdyjak.powerampwear.music_browser.artists;

import android.support.annotation.NonNull;
import android.view.View;

import com.pdyjak.powerampwear.R;
import com.pdyjak.powerampwear.music_browser.SimpleTwoLineItemViewHolderBase;

public class ArtistViewHolder extends SimpleTwoLineItemViewHolderBase {
    public ArtistViewHolder(View itemView) {
        super(itemView);
    }

    @Override
    public void bindWith(@NonNull Object object) {
        super.bindWith(object);
        ArtistItem item = (ArtistItem) object;
        mFirstLine.setText(item.name);
        mSecondLine.setText(itemView.getContext()
                .getString(R.string.songs_count_format, item.songsCount));
    }
}

File: wear/src/main/java/com/pdyjak/powerampwear/music_browser/artists/ArtistViewHolder.kt
package com.pdyjak.powerampwear.music_browser.artists

import android.view.View

import com.pdyjak.powerampwear.R
import com.pdyjak.powerampwear.music_browser.SimpleTwoLineItemViewHolderBase

class ArtistViewHolder(itemView: View) : SimpleTwoLineItemViewHolderBase(itemView) {

    override fun bindWith(obj: Any?) {
        super.bindWith(obj)
        (obj as ArtistItem).let {
            mFirstLine.text = it.name
            mSecondLine.text = itemView.context.getString(R.string.songs_count_format,
                    it.songsCount)
        }
    }
}
